README.md

```diff
@@ -65,7 +83,7 @@ run.yml
   tasks:
     - name: Network Backup and Resource Manager
```

Comment:
Can the task names be more specific to what actually is being done in the task? Something like:
"Using the network.backup validated content to perform a full device configuration backup".

Reply:
Valid for all tasks in the README.